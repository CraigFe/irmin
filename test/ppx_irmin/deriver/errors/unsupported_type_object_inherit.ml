type y = < y : unit >

type 'a t = < x : unit ; y > [@@deriving irmin]
