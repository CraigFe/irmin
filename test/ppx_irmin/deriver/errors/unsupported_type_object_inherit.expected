File "unsupported_type_object_inherit.ml", line 3, characters 25-26:
Error: ppx_irmin: inherited object type y encountered. Inheritance in object types is not supported.
